### PR TITLE
docs(appium): add landing page and resources page docs

### LINKED ISSUE #16793: appium 2 docs: landing page
`packages/appium/docs/en/intro/index.md`

### PR DESCRIPTION
**Pull request by jlipps**
closes #16793

### CODE REVIEW

File: packages/appium/docs/en/index.md [hunk 1]
```diff
@@ -1,9 +1,61 @@
...
 
-This is the Appium documentation.
+<div style="text-align: center">
+  <img src="assets/images/appium-logo-horiz.png" style="max-width: 400px;" />
+</div>
 
-TODO
+Welcome to the Appium documentation! Appium is an open-source project and ecosystem of related
+software, designed to facilitate UI automation of many app platforms, including mobile (iOS,
+Android, Tizen), browser (Chrome, Firefox, Safari), desktop (macOS, Windows), TV (Roku, tvOS,
```
Comment (KazuCocoa) [inline packages/appium/docs/en/index.md:13]: Maybe the Tizen can be in TV rather than mobile nowadays

**Review by KazuCocoa — APPROVED** [review]
otherwise, lg

File: packages/appium/docs/en/index.md (hunk 1 above)
Comment (jlipps) [reply packages/appium/docs/en/index.md:13]: i think the samsung tizen driver doesn't work on the tv, only on their mobile platform

Comment (jlipps) [reply packages/appium/docs/en/index.md:13]: but maybe the tizen mobile driver doesn't work at all in which case yeah we shouldn't mention it

**Pull request merged**